static/app/components/checkbox.tsx

```diff
@@ -1,16 +1,37 @@
 import {useEffect, useRef} from 'react';
+import styled from '@emotion/styled';
+import {useHover, usePress} from '@react-aria/interactions';
+import {mergeProps} from '@react-aria/utils';
+
+import InteractionStateLayer from 'sentry/components/interactionStateLayer';
+import {IconCheckmark, IconSubtract} from 'sentry/icons';
+import commonTheme, {FormSize} from 'sentry/utils/theme';
 
 type CheckboxProps = React.InputHTMLAttributes<HTMLInputElement>;
 
-interface Props extends Omit<CheckboxProps, 'checked'> {
+interface Props extends Omit<CheckboxProps, 'checked' | 'size'> {
   /**
    * Is the checkbox active? Supports 'indeterminate'
    */
   checked?: CheckboxProps['checked'] | 'indeterminate';
+  /**
+   *
+   */
+  size?: FormSize;
 }
 
-const Checkbox = ({checked = false, ...props}: Props) => {
+type CheckboxConfig = {borderRadius: string; box: string; icon: string};
+
+const checkboxSizeMap: Record<FormSize, CheckboxConfig> = {
```

Comment:
@vuluongj20 is this the right place for this? Or should it be in the `theme`?

Reply:
I imagine this would be part of theme, since that's where most of the other stuff is

Reply:
I think it should be here, since it's only being used here. For me it's mostly a matter of findability. If I'm looking for checkbox styles, I'd look at `checkbox.tsx` first, not `theme`.